rts/Sim/Units/Scripts/CobThread.h

```diff
@@ -147,21 +148,23 @@ class CCobThread
 	int waitAxis = -1;
 	int waitPiece = -1;
 
-	int callStackSize = 0;
-	int dataStackSize = 0;
-
 	int errorCounter = 100;
 
 	int luaArgs[MAX_LUA_COB_ARGS] = {0};
 
 
-	std::array<CallInfo, 64> callStack;
-	std::array<int, 1024> dataStack;
+	std::vector<CallInfo> callStack;
```

Comment:
Consider preventing accidents stemming from not using it like a stack:
```suggestion
	std::stack<CallInfo, std::vector<CallInfo>> callStack;
```